start.sh

```diff
@@ -3,8 +3,8 @@
 export ELASTICSEARCH=${ELASTICSEARCH:-http://172.17.42.1:9200}
 export KIBANA_INDEX=${KIBANA_INDEX:-.kibana}
 export DEFAULT_APP_ID=${DEFAULT_APP_ID:-discover}
-export REQUEST_TIMEOUT=${REQUEST_TIMEOUT:-60}
```

Comment:
It sometimes worked for me too ;) My requests were taking 58-62ms, which made this interesting to debug heh.